data/index.html

```diff
@@ -2,15 +2,29 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Hello world</title>
+    <title>Hello World</title>
     <script src="screenly.js?version=1"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://playground.srly.io/bootstrap/theme/playground-theme.css">
   </head>
   <body>
-    <p>Hi <span id="username"></span>,</p>
-    <p>I'm <span id="screen-name"></span>.</p>
+    <main>
+      <div class="container">
+        <div class="px-4 py-5 my-5 text-center">
+          <h1 class="display-5 fw-bold">Greetings <span id="greeting"></span>!</h1>
+          <p>I'm <span id="screen-name"></span>. Assuming you've pinned me in the right location, I'm located in <span id="screen-location"></span> (more precisely at latitude <span id="screen-lat"></span>&#176; and longitude <span id="screen-lng"></span>&#176;).</p>
+          <p>My Screenly ID is <span id="screen-hostname"></span> (which convenientely is also my hostname), and I'm running on a <span id="screen-hardware"></span>.</p>
+        </div>
+      </div>
+    </main>
   </body>
   <script>
-    document.getElementById("username").innerText = screenly.settings.username || "stranger";
+    document.getElementById("greeting").innerText = screenly.settings.greeting;
```

Comment:
Probably we should left `|| "stranger"` here, @vpetersson ?

Reply:
Yes. Good catch.